web/src/reducers/patch.test.js

```diff
@@ -14,8 +14,16 @@ describe('JSON patch reducer', () => {
     expect(reducer(undefined, {})).toEqual([]);
   });
 
-  it('resets to the initial state when an APD is saved', () => {
-    expect(reducer({}, { type: SAVE_APD_SUCCESS })).toEqual([]);
+  it('removes the already-saved entries when an APD is successfully saved', () => {
+    // Prime the reducer by trigger a save request. The state at this point is
+    // marked for saving and should be removed when the safe is cucessful
```

Comment:
```suggestion
    // marked for saving and should be removed when the safe is successful
```

Reply:
although that was quite an impressive typo

Reply:
What the hell? 🤣